internal/testprocessor/runs.go

```diff
@@ -0,0 +1,169 @@
+package testprocessor
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sort"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsconf "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	gconfig "github.com/gookit/config/v2"
+	"github.com/pkg/errors"
+	zlog "github.com/rs/zerolog/log"
+	"gopkg.in/yaml.v3"
+)
+
+const clickhouseDatabase = "clickhouse"
+
+type ImportRunsParams struct {
+	AwsAccessKeyID        string
+	AwsSecretAccessKey    string
+	AwsRegion             string
+	AwsQueryRunsTableName string
+
+	RunsBefore time.Time
+	RunsAfter  time.Time
+
+	OutputPath string
+}
+
+func (p *ImportRunsParams) Retrieve(_ context.Context) (aws.Credentials, error) {
+	return aws.Credentials{
+		AccessKeyID:     p.AwsAccessKeyID,
+		SecretAccessKey: p.AwsSecretAccessKey,
+		Source:          "local config",
+	}, nil
+}
+
+type RunsInput struct {
+	Runs []*Run `mapstructure:"runs" yaml:"runs"`
+}
+
+type Run struct {
+	Database    string        `mapstructure:"database" yaml:"database" dynamodbav:"Database"`
+	Version     string        `mapstructure:"version" yaml:"version" dynamodbav:"Version"`
+	Query       string        `mapstructure:"query" yaml:"query" dynamodbav:"Input"`
+	Timestamp   time.Time     `mapstructure:"timestamp" yaml:"timestamp" dynamodbav:"CreatedAt"`
+	TimeElapsed time.Duration `mapstructure:"-" yaml:"-" dynamodbav:"-"`
+}
+
+func (r *Run) CsvRow() []string {
+	return []string{r.Database, r.Version, r.Query, r.TimeElapsed.String()}
+}
+
+func loadRuns(runsFilePath string) (*RunsInput, error) {
+	err := gconfig.LoadFiles(runsFilePath)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to load runs")
+	}
+
+	runs := new(RunsInput)
+	err = gconfig.BindStruct("", runs)
+	if err != nil {
+		return nil, errors.Wrap(err, "runs data binding failed")
+	}
+
+	runs.validate()
+
+	return runs, nil
+}
+
+// validate verifies the loaded runs and sets default values for missed fields.
+func (runs *RunsInput) validate() {
+	for _, run := range runs.Runs {
+		if run.Database == "" {
+			run.Database = clickhouseDatabase
+		}
+	}
+}
+
+func ImportRunsFromAWS(params *ImportRunsParams) {
```

Comment:
At the moment we use fmt.Printxxx to report progress and zlog to report errors. These tools have different output formats. It looks inconsistent from the user's perspective.

How about using fmt.Print*** for everything as it provides more control on what you output?
Also, instead of logging errors you can return them and print them in the caller function.